### PR TITLE
chore: add installation steps to readme

### PR DESCRIPTION
**Pull request by dynamite-bud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,16 +7,54 @@ See [wasix.org](https://wasix.org) for more.
 
 ## Installation
 
-To install this Cargo subcommand, first you'll want to [install
-Rust](https://www.rust-lang.org/tools/install) and then you'll execute:
+You can install this Cargo subcommand via:
 
-```
+> Note: Installation requires [Rust](<(https://www.rust-lang.org/tools/install)>) be installed on your system.
```
Comment (theduke) [inline README.md:12]: Should probably also mention that rustup is required?

File: README.md [hunk 2]
```diff
@@ -7,16 +7,54 @@ See [wasix.org](https://wasix.org) for more.
...
+### Cargo Install
+
+```shell
 $ cargo install cargo-wasix
 ```
 
-After that you can verify it works via:
+### Cargo Binstall
+
+> Uses pre-built binaries.
```
Comment (theduke) [inline README.md:33]: Maybe link to bininstall and a quick explanation what it is?

**Review by theduke — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (dynamite-bud) [reply README.md:36]: added explaination

File: README.md (hunk 1 above)
Comment (dynamite-bud) [reply README.md]: rustup is now mentioned

**Pull request merged**